experimenter/experimenter/experiments/models.py

```diff
@@ -643,6 +646,10 @@ def treatment_branches(self):
             branches = branches.exclude(id=self.reference_branch.id)
         return list(branches)
 
+    @property
+    def is_desktop(self):
+        return self.application == self.Application.DESKTOP
```

Comment:
oh nice, we have couple of places in the code where we can use this property now